satpy/readers/viirs_vgac_l1c_nc.py

```diff
@@ -73,12 +73,6 @@ def set_time_attrs(self, data):
             self._end_time = data.attrs["end_time"]
             self._start_time = data.attrs["start_time"]
 
-    def dt64_to_datetime(self, dt64):
-        """Conversion of numpy.datetime64 to datetime objects."""
-        if isinstance(dt64, np.datetime64):
-            return dt64.astype(dt.datetime)
-        return dt64
```

Comment:
How are you able to straight remove this with no other changes in this file? Was nothing else using this?

Reply:
Yes, it was only used once in the tests